style.css

```diff
@@ -1,4 +1,4 @@
-h1 {
-  color: rgb(102, 51, 153);

```

Comment:
- [ ] Great job on making the changes requested by the previous reviewer, but there are still some things that are left to be implemented. This Pull request has a very good title but the description /summary can be better than  _**"This is a Day 2 Microverse project. And this is a pull request comment"**_  See below 👇 for an example of a descriptive summary. 

This is necessary for your future project as it helps your collaborators know exactly what features/changes you implemented on the project. 
![](https://user-images.githubusercontent.com/93039359/191243176-550fc938-97e4-4cf7-969c-ce112fd46536.png)